### PR TITLE
ConsoleInteraction: Enhance require_setting doc

### PR DESCRIPTION
**Pull request by Vedarth**
Adds docstring for `return` statement of `require_setting`.

Closes https://github.com/coala/coala/issues/4045

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by Lavenderjiang** [comment]
ready b78333ee

**Comment by Vedarth** [comment]
@Kanav-7 Sure 😄

**Comment by radonys** [comment]
ack 39a04d3a57b85fa0e8561bb41c1f780e052b49d2

File: coalib/output/ConsoleInteraction.py [hunk 1]
```diff
@@ -547,6 +547,7 @@ def require_setting(setting_name, arr, section):
                          of the bears who need this setting in [1] and
                          following.
     :param section:      The section the action corresponds to.
+    :param return:       Takes input of the missing setting from the user.
```
Comment (Makman2) [inline coalib/output/ConsoleInteraction.py]: The `return` takes input? That's kind of weird :D
Probably you want to say it returns the setting value that was requested from the user ;)

**Comment by Makman2** [comment]
Please remove the period after the issue reference.

Also:
"Fixes" is a bad word here for the shortlog, maybe "Enhance" or "Complement" or similar

**Comment by Makman2** [comment]
unack 39a04d3

**Comment by yash042** [comment]
Hello, I am completely new to Open Source and I want to contribute to coala. Can I take up this issue.

**Comment by newbazz** [comment]
Hey @yash042 this is a PR that you have commented on, read http://api.coala.io/en/latest/Developers/Newcomers_Guide.html and go ahead.

**Comment by Vedarth** [comment]
@Makman2 Alright. I am on it.
@newbazz Sorry buddy I am working on it right now

File: coalib/output/ConsoleInteraction.py (hunk 1 above)
Comment (Vedarth) [reply coalib/output/ConsoleInteraction.py]: Sorry, I didn't frame it properly 🤣. Gonna fix it asap.

File: coalib/output/ConsoleInteraction.py [hunk 2]
```diff
@@ -547,6 +547,8 @@ def require_setting(setting_name, arr, section):
                          of the bears who need this setting in [1] and
                          following.
     :param section:      The section the action corresponds to.
+    :param return:       Returns the setting value that was requested by the
```
Comment (Makman2) [inline coalib/output/ConsoleInteraction.py]: requested ~~by~~ from
:+1:

**Comment by Makman2** [comment]
unack  83d6519

File: coalib/output/ConsoleInteraction.py (hunk 2 above)
Comment (Vedarth) [reply coalib/output/ConsoleInteraction.py]: Alright 😄

**Comment by Makman2** [comment]
ack d788443

**Comment by Makman2** [comment]
reack d788443

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Pull request merged**